src/Symfony/Component/Validator/Constraints/Null.php

```diff
@@ -11,17 +11,19 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
-use Symfony\Component\Validator\Constraint;
-
-/**
+/*
```

Comment:
How do we keep the double * from triggering a coding standards warning? (as it's above an empty line and I don't think we can put the docblock right above an if statement)